web/src/containers/admin/EditAccount.js

```diff
@@ -0,0 +1,201 @@
+                name="name"
+                className="input"
+                value={name || ''}
+                onChange={this.handleEditAccount}
+              />
+            </div>
+            <div className="mb2">
+              <label htmlFor="edit_account_email">Email</label>
+              <input
+                id="edit_account_email"
+                type="text"
+                name="email"
+                className="input"
+                value={email}
+                onChange={this.handleEditAccount}
+              />
+            </div>
+            <div className="mb2">
+              <label htmlFor="edit_account_phone">Phone number</label>
+              <input
+                id="edit_account_phone"
+                type="text"
+                name="phone"
+                className="input"
+                value={phone || ''}
+                onChange={this.handleEditAccount}
+              />
+            </div>
+            <div className="mb2">
+              <label htmlFor="edit_account_position">Position</label>
+              <input
+                id="edit_account_position"
+                type="text"
+                name="position"
+                className="input"
+                value={position || ''}
+                onChange={this.handleEditAccount}
+              />
+            </div>
+            <div className="mb2">
+              <label htmlFor="edit_account_state">State</label>
+              <select
+                id="edit_account_state"
+                name="state"
+                className="input"
+                value={state || ''}
+                onChange={this.handleEditAccount}
+              >
+                <option value="">None</option>
+                {STATES.map(s => (
+                  <option key={s.id} value={s.id}>
+                    {s.name}
+                  </option>
+                ))}
+              </select>
+            </div>
+            <div className="mb2">
+              <label htmlFor="edit_account_role">Authorization role</label>
+              <select
+                id="edit_account_role"
+                name="role"
+                className="input"
+                value={role || ''}
+                disabled={currentUser.id === user.id}
+                onChange={this.handleEditAccount}
+              >
+                <option value="">None</option>
+                {roles.map(r => (
+                  <option key={r.name} value={r.name}>
+                    {r.name}
+                  </option>
+                ))}
+              </select>
+            </div>
+            <Btn type="submit" disabled={fetching}>
+              {fetching ? 'Working' : 'Edit account'}
+            </Btn>
+          </form>
+        </div>
+      );
+    }
+    return null;
+  };
+
+  handlePickAccount = e => {
+    const { users } = this.props;
+    const { value } = e.target;
+
+    const user = users.filter(u => u.id === +value).reduce((_, u) => u, null);
```

Comment:
IE doesn't support [Array.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find), so this seems like a reasonable workaround - if there's not a matching user, returns `null`; else returns the single matching user.